Assignment-3/src/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>3rd-Responsive</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <h1>CSS Color Markers</h1>
+
+    <div class="container">
+
+        <div class="marker red">
+
+            <div class="cap"></div>
+
+            <div class="sleeve"></div>
+
+        </div>
+
+        <div class="marker green">
+
+            <div class="cap"></div>
+
+            <div class="sleeve"></div>
+
+        </div>
+
+        <div class="marker blue">
+
+            <div class="cap"></div>
+
+            <div class="sleeve"></div>
+
```

Comment:
We need to reomve these unnecessary empty lines from this and all other assignments.